scope/src/doctor/check.rs

```diff
@@ -225,6 +226,21 @@ impl DefaultDoctorActionRun {
         Ok(capture.exit_code.unwrap_or(-1))
     }
 
+    fn generate_env_vars(&self) -> BTreeMap<String, String> {
+        let mut env_vars = BTreeMap::new();
+        env_vars.insert(
+            "SCOPE_BIN_DIR".to_string(),
+            std::env::current_exe()
+                .unwrap()
```

Comment:
nit: I think it would be better in general to use `expect("some string")` to the panic is useful, but in this case probably not important.